recipes/threadpoolctl/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: threadpoolctl
+  version: {{ version }}
+
+source:
+  url: https://github.com/joblib/threadpoolctl/archive/{{ version }}.tar.gz
+  sha256: 49b987e9e86168eec68d0c706091bd6a8590268abc249b32bcd2a3909d331a04
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m flit install
+
+requirements:
+  host:
+    - python >=3.5
+    - flit
+  run:
+    - python >=3.5
+
+test:
+  requires:
+    - numpy
+  imports:
+    - threadpoolctl
+  commands:
+    - python -c "import numpy; from threadpoolctl import threadpool_info; threadpool_info()"
+    - python -c "import numpy; from threadpoolctl import threadpool_limits; threadpool_limits(1)"
+    - python -m threadpoolctl -i numpy
+
+about:
+  home: https://github.com/joblib/threadpoolctl
+  license: BSD
```

Which version of BSD is this?

---

Looks like [BSD-3-Clause]( https://github.com/joblib/threadpoolctl/blob/master/LICENSE ). Can you please confirm/deny @jeremiedbb and if so commit this (or other appropriate) change? 🙂

```suggestion
  license: BSD-3-Clause
```

---

it is. I updated.